### ISSUE TITLE
Problems with the uuu script for 6ull

### ISSUE THREAD

**Issue by shun12a** [nxp-imx/mfgtools#401]
Does mx6ull support uuu script FBK?
I made a comparison between FB and FBK and found that FB could get feedback while FBK could not. I also found on NXP's official website what NXP employees said: "seems fbk is working only with android as described in", If  I want to use FBK, I get stuck in Wait for Known USB.Thank you for your advice

**Comment by shun12a** [comment]
![image](https://github.com/nxp-imx/mfgtools/assets/96690717/c7eaba00-146d-4c1a-87f6-9cd5fd0f6e58)

**Comment by shun12a** [comment]
![image](https://github.com/nxp-imx/mfgtools/assets/96690717/4cd7ad2f-a6f3-4dd0-8e29-6f8be77562f8)

**Comment by nxpfrankli** [comment]
Make sure your initramfs included new https://github.com/nxp-imx/imx-uuc

**Comment by shun12a** [comment]
> Make sure your initramfs included new https://github.com/nxp-imx/imx-uuc
Yes, thank you for your answer. How can I replace it? I found that the official "fsl-image-mfgtool-initramfs-imx_mfgtools.cpio.gz.u-boot" does not have a C file about uuc. Do I still need to mount it?

**Comment by shun12a** [comment]
![image](https://github.com/nxp-imx/mfgtools/assets/96690717/ca1ae30a-2443-4719-bae4-26282b8114e9)

**Comment by shun12a** [comment]
thank for your help,I was so inspired by your answer

**Comment by shun12a** [comment]
I have another question about how wic is created. I want to use their own compiled files to make wic files burned to the development board

**Comment by shun12a** [comment]
![image](https://github.com/nxp-imx/mfgtools/assets/96690717/894723ab-8954-4068-a8f9-22f6c92b1184)

**Comment by shun12a** [comment]
![image](https://github.com/nxp-imx/mfgtools/assets/96690717/09b1aa49-4c37-429c-b7f8-98adbe0ae904)

**Comment by shun12a** [comment]
> Make sure your initramfs included new https://github.com/nxp-imx/imx-uuc

cool

**Comment by nxpfrankli** [comment]
Long time no update, close it
